pageserver/src/tenant.rs

```diff
@@ -1000,11 +1003,16 @@ impl Tenant {
                 // Remote preload is complete.
                 drop(remote_load_completion);
 
+
                 // We will time the duration of the attach phase unless this is a creation (attach will do no work)
+                let attach_start = std::time::Instant::now();
                 let attached = {
                     let _attach_timer = Some(TENANT.attach.start_timer());
                     tenant_clone.attach(preload, &ctx).await
                 };
+                let attach_duration = attach_start.elapsed();
+                let wal_lag_cooldown = attach_start + 2 * attach_duration + Duration::from_secs(60);
+                _ = tenant_clone.attach_wal_lag_cooldown.set(wal_lag_cooldown);
```

Comment:
`tenant_clone.attach_wal_lag_cooldown.set(wal_lag_cooldown).ok();`?

Reply:
From what I know, the standard way to throw away results is `let _ =` and `_  =`.